timeseries/src/autogluon/timeseries/metrics/__init__.py

```diff
@@ -2,7 +2,7 @@
 from typing import Type, Union
```

Comment:
Should we add a unittest to ensure that our implementation doesn't break over time (similar to https://github.com/autogluon/autogluon/blob/master/timeseries/tests/unittests/test_metrics.py#L219)?